asapdiscovery-data/asapdiscovery/data/schema_v2/ligand.py

```diff
@@ -312,3 +316,27 @@ def compound_names_unique(ligands: list[Ligand]) -> bool:
     """
     compound_names = [ligand.compound_name for ligand in ligands]
     return len(set(compound_names)) == len(compound_names)
+
+
+def write_ligands_to_multi_sdf(
```

Comment:
Nice I think we will end up with a few of these types of functions which deal with collections of molecules such as deduplication and loading sets of ligands etc maybe it should have its own file like `ligand_utils` or just `utils`?

Reply:
I might leave it there for now, but can move in future, when more crud arrives.